confectory-core/src/test/java/net/obvj/confectory/ConfigurationContainerTest.java

```diff
@@ -198,6 +198,15 @@ void getStringProperty_keyOnlyAndLenient_configurationWithoutNamespace()
         assertThat(container.getStringProperty(KEY_STRING), equalTo("string2")); // CONF_NS1_PROPERTIES_2 (lenient)
     }
 
+    @Test
+    void getUnsortedStringProperty_keyOnlyAndLenient_configurationWithoutNamespace() {
+        container = new ConfigurationContainer(DataFetchStrategy.LENIENT_UNSORTED, CONF_NS1_PROPERTIES_1, CONF_NS1_PROPERTIES_2,
+                CONF_NS2_PROPERTIES_1, CONF_PROPERTIES_1);
+        assertThat(container.getStringProperty(KEY_TEST), equalTo("ok01")); // CONF_NS2_PROPERTIES_1 (lenient)
+        String responseValue = container.getStringProperty(KEY_STRING); 
+        assertTrue(responseValue.equals("string1") || responseValue.equals("string2"));
```

Comment:
Always prefer to use `assertThat` which is more readable and better for troubleshooting:
`assertThat(container.getStringProperty(KEY_STRING), ... );